coordinator/cmd/api/app/mock_app.go

```diff
@@ -90,17 +90,9 @@ func (c *CoordinatorApp) MockConfig(store bool) error {
 	cfg.ProverManager = &coordinatorConfig.ProverManager{
 		ProversPerSession: 1,
 		Verifier: &coordinatorConfig.VerifierConfig{
-			MockMode: true,
```

Comment:
why remove this line

Reply:
The unit test will enable "mock_verifier" by `-tags="mock_verifier"`, this code is for the real verifier and seems never to have been used.